### ISSUE TITLE
Als Product Owner wil ik dat de inwoner weet dat een reservering/melding alleen met DigiD kan gebeuren.

### ISSUE THREAD

**Issue by RickWildHoorn** [VNG-Realisatie/huwelijksplanner#232]

**Comment by CMasselink** [comment]
@JeroenduC Zou jij naar bovenstaande willen kijken?

**Comment by JeroenduC** [comment]
Misschien is het een idee om te kijken hoe dit nu gaat met de formulieren van de gemeente Utrecht? 

Stap 1 - Uitleg formulier, wat moet er gaan gebeuren en dat je dat met DigiD moet gaan doen.
![image](https://user-images.githubusercontent.com/49227194/77668780-2710cc80-6f84-11ea-8091-7ed0734341b6.png)


Stap 2 - Daadwerkelijk inloggen (stap voor stap) en de uitleg er bij.
![image](https://user-images.githubusercontent.com/49227194/77668737-17918380-6f84-11ea-8758-d9228acbe07b.png)

Zoiets?

**Comment by RickWildHoorn** [comment]
Ja! Alleen moeten we dan wel even kijken hoe dit werkt i.c.m. het feit dat men ook alleen kan 'orienteren/rondkijken', zonder dat ze meteen inloggen.

**Comment by rubenvdlinde** [comment]
@bbrands02 zou je bovenstaande vormgeving alvast willen toevoegen aan de UI ?

@RickWildHoorn volgensmij is er al ergens een los issue voor de discussie rondom rondkijken?

**Comment by RickWildHoorn** [comment]
Ja dat is #310 .

**Comment by MichelleDautzenberg** [comment]
Er zou dus ook een korte tekst moeten staan op de pagina voor uitnodigen partner, getuigen en eigen Babs dat zij alleen met Digid kunnen instemmen. 
En op de hoofdpagina dat je als koppel alleen kan starten met DigiD.

**Comment by JeroenduC** [comment]
Voorstel: 
Partner: "Om te bevestigen heeft uw partner DigiD nodig."
Getuigen: "Om te bevestigen heeft uw getuige DigiD nodig."
